docs/control-structures.rst

```diff
@@ -113,8 +113,8 @@ actual contract has not been created yet.
 Functions of other contracts have to be called externally. For an external call,
 all function arguments have to be copied to memory.
 
-When calling functions
-of other contracts, the amount of Wei sent with the call and the gas can be specified::
+When calling functions of other contracts, the amount of Wei sent with the call and
+the gas can be specified with special modifiers ``.value()`` and ``.gas()``, respectively::
```

Comment:
I think we should also mention that `value` is only available for `payable` functions.